driver_test.go

```diff
@@ -5,15 +5,28 @@ import (
 )
 
 func TestDriver(t *testing.T) {
-	var d Driver = NewNoopDriver()
+	var f DriverFactory = NoopFactory()
+	var params []ConfigParameter = []ConfigParameter{{
+		Name:  "Sample Parameter",
+		Value: "Value",
+	}}
+
+	temp, err := f.CreateDriver(params, nil)
+	d := temp.(Driver)
```

Comment:
Explicit cast required by consumer since the interface returns interface{}.  I haven't come up with a better alternative yet, but I'm open to feedback.